examples/bench_call.rs

```diff
@@ -0,0 +1,64 @@
+fn main() {
+  let platform = v8::new_default_platform(0, false).make_shared();
+  v8::V8::initialize_platform(platform);
+  v8::V8::initialize();
+  let isolate = &mut v8::Isolate::new(v8::CreateParams::default());
+  let handle_scope = &mut v8::HandleScope::new(isolate);
+  let context = v8::Context::new(handle_scope);
+  let scope = &mut v8::ContextScope::new(handle_scope, context);
+  let global = context.global(scope);
+
+  extern "C" fn callback(info: *const v8::FunctionCallbackInfo) {
+    let scope = unsafe { &mut v8::CallbackScope::new(&*info) };
+    // let args = unsafe { v8::FunctionCallbackArguments::from_function_callback_info(info) };
+    let mut rv = unsafe { v8::ReturnValue::from_function_callback_info(info) };
+    rv.set(v8::Integer::new(scope, 42).into());
+  }
+  let func = v8::Function::new_raw(scope, callback).unwrap();
+  let name = v8::String::new(scope, "function_new_raw").unwrap();
+  global.set(scope, name.into(), func.into()).unwrap();
+
+  let func = v8::Function::new(
+    scope,
+    |scope: &mut v8::HandleScope,
+     _: v8::FunctionCallbackArguments,
+     mut rv: v8::ReturnValue| {
+      rv.set(v8::Integer::new(scope, 42).into());
+    },
```

Comment:
You could use the `rv.set_u32(42)`, if that is not contrary to the benchmark.

Reply:
That does speed up things quite a bit (~15%). Unfortunately not sure if we can do that with ops in Deno..

Reply:
The ops macro does have a special case for u32 assigning and it would be possible to add all the other special cases as well (bool etc.)

Reply:
^ its optimized AOT by the ops macro